### PR TITLE
(MP3) Command Courtyard video

### PR DESCRIPTION
**Pull request by MasierAzulite**
Replaced the video linked for Command Courtyard with a more accurate video.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -59,6 +59,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Changed: When Progressive Grapple is enabled, it will now show `2 shuffled copies` rather than `Shuffled` for better consistency.
 - Changed: A proper error message is displayed when mono is not found, when exporting a game on macOS and Linux.
 
+### Metroid Prime 3: Corruption
+
+#### Logic Database
+
+- Added: replaced the video linked for Command Courtyard with a more accurate video.
```
Comment (Miepee) [inline CHANGELOG.md:66]: "Changed" instead of "Added" seems more appropriate for this

Comment (dyceron) [reply CHANGELOG.md:66]: We don't need changelogs for Prime 3

Comment (MasierAzulite) [reply CHANGELOG.md]: Felt weird adding one, but I just did what I was taught haha. Deleted.

File: CHANGELOG.md
Comment (Miepee) [inline CHANGELOG.md:1]: I dont think you meant to delete *the entire* changelog lmao

File: CHANGELOG.md [hunk 2]
```diff
@@ -59,6 +59,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Changed: When Progressive Grapple is enabled, it will now show `2 shuffled copies` rather than `Shuffled` for better consistency.
 - Changed: A proper error message is displayed when mono is not found, when exporting a game on macOS and Linux.
 
+
+
```
Comment (dyceron) [inline CHANGELOG.md:63]: remove these extra lines

**Review by dyceron — CHANGES_REQUESTED** [review]

**Review by dyceron — APPROVED** [review]

**Pull request merged**